libraries/ESP8266WiFiMesh/examples/HelloMesh/HelloMesh.ino

```diff
@@ -1,50 +1,129 @@
 #include <ESP8266WiFi.h>
 #include <ESP8266WiFiMesh.h>
 
+String mesh_name = "Mesh_Node";
```

Comment:
This should probably be declared as String meshName(F("meshNode"));
Also, please use camelCase for naming convention.

Reply:
Camel case or snake case work either way for me. I noticed snake case was used in the original mesh library, but I take it style conventions has changed since then?

Regarding the actual mesh name, though, I think it may be prudent to not write it in the same style as a programming object. It is just a network name, after all, and its lack of relationship to other program objects may be easier for the novice user to deduce when it does not follow the typical naming conventions for objects in the library. Thus `String meshName(F("Mesh_Node"));` or `String meshName(F("MeshNode_"));` or some such mesh name format may be useful.

Reply:
Style wasn't really pursued before, commenting on it is rather recent. In general, current naming convention is camelCase for C++ and snake_case for C.